packages/fxa-auth-server/lib/geocoding.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Client,
+  PlaceType2,
+  Status,
+} from '@googlemaps/google-maps-services-js';
+import countries from 'i18n-iso-countries';
+import { ConfigType } from '../config';
```

Comment:
The TS Hero plugin has a feature that sorts the imports correctly in VS Code, just hit cmd-shift-p, then type 'ts her' and it should show the Organize Imports function.

Reply:
Edit: After talking with Ben about this "organize imports on save" option for the TS Hero extension, the downside of that is that it will also remove unused imports, which can be annoying during development. An alternative is to use the command palette (cmd + shift + P on Mac) and select "TS Hero: Organize imports".